core/codegen/src/lib.rs

```diff
@@ -774,6 +774,28 @@ pub fn derive_from_form(input: TokenStream) -> TokenStream {
     emit!(derive::from_form::derive_from_form(input))
 }
 
+/// Derive for the [`FromParam`] trait.
+///
+/// The [`FromParam`] derive can be applied to enums with nullary
+/// (zero-length) fields:
```

Comment:
Add `/// [`FromParam`]: ../rocket/request/trait.FromParam.html` to this doc comment, so these links will link to the trait.